spec/controllers/search_references_controller_spec.rb

```diff
@@ -27,4 +27,9 @@
     expect(response.body).to include 'Machine Tools'
     expect(response.body).to include '/sections/16'
   end
+
+  it 'doesn\'t renders links to commodities' do
+    expect(response.body).not_to include 'Mascarpone'
+    expect(response.body).not_to include '/commodities/0406105090'
+  end
```

Comment:
I know it's a pattern in this codebase, but I think having multiple expectations inside the same block is not recommended. Did Rubocop not complain about this?